api/views/purchaseimport.py

```diff
@@ -161,9 +162,18 @@ def _create_purchase_for_canteen(self, siret, row):
         date = row.pop(0)
         if date == "":
             raise ValidationError({"date": "La date ne peut pas être vide"})
-        price = row.pop(0)
+
+        price = row.pop(0).strip()
         if price == "":
             raise ValidationError({"price_ht": "Le prix ne peut pas être vide"})
+
+        # We try to round the price. If we can't, we will let Django's field validation
+        # manage the error - hence the `pass` in the exception handler
+        try:
+            price = Decimal(price.strip()).quantize(Decimal(".01"), rounding=ROUND_HALF_DOWN)
```

Comment:
```suggestion
            price = Decimal(price).quantize(Decimal(".01"), rounding=ROUND_HALF_DOWN)
```
il y a déjà le strip sur 166